.container_app/container_app.tf

```diff
@@ -0,0 +1,87 @@
+data "azurerm_resource_group" "resource_group_app" {
+  name = "${local.project}-container-app-rg"
+}
+
+data "azurerm_key_vault" "key_vault" {
+  resource_group_name = var.key_vault.resource_group_name
+  name                = var.key_vault.name
+}
+
+data "azurerm_key_vault_secrets" "key_vault_secrets" {
+  key_vault_id = data.azurerm_key_vault.key_vault.id
+}
+
+data "azurerm_key_vault_secret" "keyvault_secret" {
+  for_each     = toset(data.azurerm_key_vault_secrets.key_vault_secrets.names)
+  name         = each.key
+  key_vault_id = data.azurerm_key_vault.key_vault.id
+}
+
+data "azurerm_container_app_environment" "container_app_environment" {
+  resource_group_name = data.azurerm_resource_group.resource_group_app.name
+  name                = "${local.project}-cae"
+}
+
+resource "azapi_resource" "container_app_onboarding" {
+  type      = "Microsoft.App/containerApps@2023-05-01"
+  name      = "${local.project}-${local.app_name}-ca"
+  location  = data.azurerm_resource_group.resource_group_app.location
+  parent_id = data.azurerm_resource_group.resource_group_app.id
+
+  tags = var.tags
+
+  identity {
+    type = "SystemAssigned"
+  }
+
+  body = jsonencode({
+    properties = {
+      configuration = {
+        activeRevisionsMode = "Single"
+        ingress = {
+          allowInsecure = true
+          external      = true
+          traffic = [
+            {
+              latestRevision = true
+              label          = "latest"
+              weight         = 100
+
+            }
+          ]
+          targetPort = 8080
+        }
+        secrets = local.secrets
+      }
+      environmentId = data.azurerm_container_app_environment.container_app_environment.id
+      template = {
+        containers = [
+          {
+            env   = concat(var.app_settings, local.secrets_env)
+            image = "ghcr.io/pagopa/selfcare-onboarding:${var.image_tag}"
+            name  = "${local.project}-${local.app_name}"
```

Comment:
this repo represents a monorepo of onboarding domain, it may contains more container apps....so it might be better identify this container app with submodule's name, in this case onboarding-ms